packaging/src/docker/5.x/.env

```diff
@@ -0,0 +1,2 @@
+ALFRESCO_TAG=5.2.3-SNAPSHOT
```

Comment:
Versions will be updated to the latest ACS version as a part of: https://issues.alfresco.com/jira/browse/SEARCH-1078, so merging this PR as is now.